tests/Basic/Basic.h

```diff
@@ -760,15 +760,10 @@ class TestNamingAnonymousTypesInUnion
 };
 
 #define CS_API
-class CS_API ClassPassTry
-{
-public:
-	int n;
-	char c;
-};
+class CS_API RefTypeClassPassTry { };
```

Comment:
#438 .. I don't think the main problem was due to CS_API. It was just that the one who filed the issue said so. But, the main problem was wrong code generation.